gulp-tasks/utils/build-browser-bundle.js

```diff
@@ -85,9 +90,9 @@ module.exports = (packagePath, buildType) => {
     prefix = '';
   }
 
-  const exports = pkgJson.workbox.disabledNamedExports ? 'default' : 'named';
-  const namespace =
-    `${prefix}${pkgJson.workbox.browserNamespace}`;
+  const namespace = pkgJson.workbox.browserNamespace ?
+      `${prefix}${pkgJson.workbox.browserNamespace}` : null;
+  const exports = namespace ? 'named' : 'none';
```

Comment:
This shouldn't be needed now since everything for browser is under the default export which means rollup will export without any naming (i.e. none).